.github/workflows/ci-linux.yaml

```
@@ -56,7 +56,7 @@ jobs:
       id: cache
       with:
         path: ${{ matrix.prefix }}
-        key: ${{ matrix.label }}-conda-${{ hashFiles('pypsa-earth/envs/environment.yaml') }}-${{ env.DATE }}-${{ env.CACHE_NUMBER }}
+        key: ${{ matrix.label }}-conda-${{ hashFiles('pypsa-distribution/envs/environment.yaml') }}-${{ env.DATE }}-${{ env.CACHE_NUMBER }}
```

This should probably do the trick :)
```suggestion
        key: ${{ matrix.label }}-conda-${{ hashFiles('envs/environment.yaml') }}-${{ env.DATE }}-${{ env.CACHE_NUMBER }}
```

---

@Margherita-Capitani we missed another path to be revised:
`mamba env update -n pypsa-earth -f pypsa-earth/envs/environment.yaml` the "pypsa-earth/" shall be dropped.